Documentation/ceph-csi-drivers.md

```diff
@@ -270,8 +270,12 @@ In short, as the documentation describes it:
 
 In [snapshotClass](cluster/examples/kubernetes/ceph/csi/example/rbd/snapshotclass.yaml),
 the `csi.storage.k8s.io/snapshotter-secret-name` parameter should reference the
-name of the secret created for the rbdplugin. The `monitors` are a comma
-separated list of your Ceph monitors and `pool` to reflect the Ceph pool name.
+name of the secret created for the rbdplugin and `pool` to reflect the Ceph pool name.
+
+Update the value of the `clusterID` field to match the namespace that rook is
```

Comment:
I think it's sufficient to say the `clusterID` should match the namespace. We don't need to mention the implementation detail about the configmap, right? I'd say we can remove all mention of the configmap from this doc.

Reply:
removed the info related to configmap